src/buffer/channel.rs

```diff
@@ -34,6 +34,11 @@ pub fn view<'a>(
     let input = history.input(&buffer);
     let our_nick = clients.nickname(&state.server);
 
+    let users = clients.get_channel_users(&state.server, &state.channel);
+    let our_user = users
+        .iter()
+        .find(|user| user.nickname() == our_nick.unwrap());
```

Comment:
We have a clients.resolve_attributes you can use

Reply:
Thanks for the heads up!